File: Lab6+7/LogReport.php
<?php


use Cassandra\Date;

class LogReport implements JsonSerializable {
    private string $type;
    private string $severity;
    private string $date;
    private string $user;
    private string $log;
    /**
     * LogReport constructor.
     * @param $type
     * @param $severity
     * @param $date
     * @param $user
     * @param $log
     */
    public function __construct($type, $severity, $date, $user, $log) {
        $this->type = $type;
        $this->severity = $severity;
        $this->date = $date;
        $this->user = $user;
        $this->log = $log;
    }

    /**
     * @return string
     */
    public function getType(): string {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType(string $type): void {
        $this->type = $type;
    }

    /**
     * @return string
     */
    public function getSeverity(): string {
        return $this->severity;
    }

    /**
     * @param string $severity
     */
    public function setSeverity(string $severity): void {
        $this->severity = $severity;
    }

    /**
     * @return string
     */
    public function getDate(): string {
        return $this->date;
    }

    /**
     * @param string $date
     */
    public function setDate(string $date): void {
        $this->date = $date;
    }

    /**
     * @return string
     */
    public function getUser(): string {
        return $this->user;
    }

    /**
     * @param string $user
     */
    public function setUser(string $user): void {
        $this->user = $user;
    }

    /**
     * @return string
     */
    public function getLog(): string {
        return $this->log;
    }

    /**
     * @param string $log
     */
    public function setLog(string $log): void {
        $this->log = $log;
    }


    public function jsonSerialize() {
        return get_object_vars($this);
    }
}

File: Lab9/Lab9/Models/LogReport.cs

namespace Lab9.Models {
    public class LogReport {
        private int _id;
        private string _type;
        private string _severity;
        private string _date;
        private string _username;
        private string _log;

        public LogReport(string type, string severity, string date, string username, string log) {
            this._type = type;
            this._severity = severity;
            this._date = date;
            this._username = username;
            this._log = log;
        }

        public LogReport(int id, string type, string severity, string date, string username, string log) {
            this._id = id;
            this._type = type;
            this._severity = severity;
            this._date = date;
            this._username = username;
            this._log = log;
        }
        public int ID {
            get => _id;
        }
        public string Type {
            get => _type;
            set => _type = value;
        }

        public string Severity {
            get => _severity;
            set => _severity = value;
        }

        public string Date {
            get => _date;
            set => _date = value;
        }

        public string Username {
            get => _username;
            set => _username = value;
        }

        public string Log {
            get => _log;
            set => _log = value;
        }
    }
}